editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -120,6 +120,18 @@ HelpBrowser::HelpBrowser( QWidget * parent ):
     setFocusProxy(mWebView);
 }
 
+void HelpBrowser::setServerPort(int serverPort)
+{
+    mServerPort = serverPort;
+}
+
+void HelpBrowser::onPageLoad()
+{
+    if (mServerPort) {
```

Comment:
personal style gripe -- I really don't like implicitly converting int (or anything) to bool in a condition. if the type of the object isn't obvious, you have to look it up to check the logic, and it also doesn't work semantically, for me anyway. would rather see this as `mServerPort != 0`. don't need to change it now, though, there are more important things in this PR